src/ipc/ipc4/handler.c

```diff
@@ -927,7 +1020,8 @@ static int ipc4_get_large_config_module_instance(struct ipc4_message_request *ip
 
 	if (ret < 0)
 		return IPC4_FAILURE;
-
+	dcache_invalidate_region((__sparse_force void __sparse_cache *)MAILBOX_HOSTBOX_BASE,
+				 config.extension.r.data_off_size);
```

Comment:
config_get now uses payload, so we need to invalidate hostbox cache here. Only vendor config get uses payload for now , so we could move this to the vendor_config_get function